### PR TITLE
feat: LT-886 - Implement form back button, and constants

### PR DESCRIPTION
**Pull request by Mark-Spiers-kainos**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [submodule "lite-content"]
-	path = lite-content
+	path = lite_content
```
Comment (andytaylorkainos) [inline .gitmodules]: thank you!

File: ui_automation_tests/pages/organisations_form_page.py [hunk 2]
```diff
@@ -58,5 +58,5 @@ def enter_last_name(self, text):
         self.driver.find_element_by_id("user.last_name").send_keys(text)
 
     def click_submit(self):
-        submit = self.driver.find_element_by_xpath("//*[@action='submit']")
+        submit = self.driver.find_element_by_css_selector("button[value='submit']")
```
Comment (andytaylorkainos) [inline ui_automation_tests/pages/organisations_form_page.py]: single quotations please (the inverse of what youve done)

File: core/builtins/custom_tags.py [hunk 3]
```diff
@@ -10,10 +10,19 @@
 from conf.constants import ISO8601_FMT
 from conf.settings import env
 from core import strings
+from lite_content.lite_internal_frontend import constants
 
 register = template.Library()
 
 
+@register.simple_tag(name="lcs")
```
Comment (andytaylorkainos) [inline core/builtins/custom_tags.py]: single quotes please

Comment (efahy) [reply core/builtins/custom_tags.py]: Done

File: queues/forms.py [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
 
 from core.builtins.custom_tags import get_string
 
-form = Form(title=get_string('queues.queue_add.page_heading'), questions=[
-    TextInput(title=get_string('queues.queue_add.question_title'),
-              name='name'),
-])
+form = Form(title=get_string('queues.queue_add.page_heading'),
+            questions=[TextInput(title=get_string('queues.queue_add.question_title'), name='name')],
+            back_link=BackLink('Back to queues', reverse_lazy('queues:queues')))
```
Comment (stephanieboyd) [inline queues/forms.py]: Should be in constants?

**Pull request merged**